templates/todo/projects/nodejs-mongo/.repo/terraform/README.md

```diff
@@ -39,6 +39,8 @@ The following prerequisites are required to use this application. Please ensure
 - [Node.js with npm (16.13.1+)](https://nodejs.org/) - for API backend and Web frontend
 - [Terraform CLI](https://aka.ms/azure-dev/terraform-install)
 
+> Note: Terraform is in **alpha mode**. You need to enable it by running `azd config set alpha.terraform on`. Read more about [alpha features](https://github.com/Azure/azure-dev/tree/main/cli/azd/docs).
```

Comment:
I think we should also update the pipeline definitions for our terraform templates to set this, otherwise in CI the feature will fail to be enabled.

Reply:
done